ui/pages/first-time-flow/select-action/index.scss

```
@@ -19,8 +19,9 @@
   }
 
   &__select-buttons {
-    display: flex;
-    flex-direction: row;
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    gap: 22px;
```

I realize this is replacing the existing value but it might be a good time to update it to adhere to the `8px` grid that we know adopt

```suggestion
    gap: 24px;
```